app/models/manageiq/providers/openstack/cloud_manager.rb

```diff
@@ -78,7 +78,7 @@ def ensure_network_manager
 
   def ensure_cinder_manager
     return false if cinder_manager
-    build_cinder_manager(:type => 'ManageIQ::Providers::StorageManager::CinderManager')
+    build_cinder_manager(:type => 'ManageIQ::Providers::Openstack::StorageManager::CinderManager')
```

Comment:
hm I don't think we can easily change the STI subclass of the manager now, since we would need a migrations that will change existing  records. Can we keep the old one? Since we need to backport this.

We can then do upstream only patch changing this, with migration.

Reply:
I had a feeling that might be an issue. So to clarify, you'd recommend just leaving this line as `build_cinder_manager(:type => 'ManageIQ::Providers::StorageManager::CinderManager')` and then changing it later along with a migration in another patch?

Reply:
yes. Since we can't backport the migration now, that would be only for upstream and future releases